scenes/game/local_game.gd

```diff
@@ -1386,6 +1395,7 @@ class Player:
 		arena_location = new_arena_location
 		events += [parent.create_event(Enums.EventType.EventType_Move, my_id, new_arena_location, "move", distance, previous_location)]
 		if position_changed:
+			moved_self_this_strike = true
```

Comment:
Would this not go within on_position_changed?
Maybe just add a param for is_self_move if the point is not to include push/pull, maybe it'll be useful for something else later.